index.html

```diff
@@ -6,6 +6,11 @@
     <title>Les 3 - Opdracht CSS Grid</title>
 </head>
 <body>
-
+<div class="gridItem gridA">A </div>
+<div class="gridItem gridB">B Lorem ipsum dolor sit amet, consectetur adipisicing elit. A nisi quidem quo quos reprehenderit! Blanditiis culpa doloremque incidunt laboriosam minus officiis quod repellendus, velit? Alias aliquam beatae delectus dolor ducimus enim, eum eveniet, explicabo facere hic illum in nihil non possimus quaerat quo quos. Accusamus amet aut commodi dignissimos distinctio ducimus eaque earum harum in ipsum magni molestias odit officia provident quidem ratione, recusandae rem soluta totam ut vitae voluptatem voluptates voluptatum! Consequuntur dolor, in itaque nemo quod sequi voluptas? Atque fuga illo, nulla porro similique unde velit. Consequuntur culpa cum deserunt dolorem error ex in ipsum laboriosam maxime modi mollitia nihil, porro provident quia quisquam quos repudiandae voluptas! A animi blanditiis commodi corporis cupiditate doloremque ea, excepturi, exercitationem expedita fuga illum in ipsam laboriosam laudantium magnam mollitia nemo nihil nostrum quis quisquam quo reprehenderit saepe ut? Doloremque, nesciunt saepe? Assumenda autem eius fuga harum id ipsum quisquam recusandae veniam?</div>
```

Comment:
Tip!: Mocht je je Lorem in een alinea willen weergeven, ga je in Webstorm naar view, kies active editor, kies vervolgens softwrap.